xarray/core/variable.py

```diff
@@ -1542,7 +1542,7 @@ def rolling_window(self, dim, window, window_dim, center=False):
         else:
             pads = (window - 1, 0)
 
-        fill_value = False if self.dtype.kind == 'b' else None
+        fill_value = False if self.dtype.kind == 'b' else dtypes.NA
```

Comment:
Maybe we should add the `fill_value` argument to this function?

I suspect special logic for boolean dypes would be surprising to many users.